modules/network-modify-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+// * networking/configuring-network-operator.adoc
+
+[id="network-modifying-config-{context}"]
+= Modifying network configuration in a running cluster
+
+The Network Operator reconciles the state of the cluster against a desired
+configuration.
+
+You can modify only `kubeProxy` configuration parameters in a running cluster.
+
+.Prerequisites
+
+* `oc` command installed.
+* Logged in to a running cluster.
+
+.Procedure
+
+. Use the following command to edit the `Network.operator.openshift.io` Custom
+Resource (CR):
++
+----
+$ oc edit network
+----
++
+The following example output shows a `Network.operator.openshift.io` CR that has
+been configured with default values.
++
+[source,yaml]
+----
+# Please edit the object below. Lines beginning with a '#' will be ignored,
+# and an empty file will abort the edit. If an error occurs while saving this file will be
+# reopened with the relevant failures.
+#
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  creationTimestamp: 2019-04-02T17:19:32Z
```

Comment:
remove the creationTimestamp, generation, resourceVersion, selfLink, and uid - those are internal k8s fields.

Reply:
@openshift/team-documentation, are we stripping out internal fields from `metadata` in general? I included it because it's verbatim output from a command.